Packs/JSONSampler/Integrations/JSONSampler/JSONSampler.yml

```diff
@@ -0,0 +1,51 @@
+category: Utilities
+commonfields:
+  id: JSONSampler
+  version: -1
+configuration:
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- defaultvalue: "1"
+  display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  required: false
+  type: 19
+- display: The raw JSON string to use as the sample data
+  name: JSON
+  required: true
+  type: 12
+- display: The incident name to give to the created incident
+  name: name
+  required: false
+  type: 0
+description: |-
+  A utility for testing incident fetching with mock JSON data.
+display: JSONSampler
+name: JSONSampler
```

Comment:
i'm thinking if we can improve the name here a bit
we already have an integration named `Sample Incident Generator` so perhaps we should go with `JSON Sample Incident Generator`
what do you think?

Reply:
That sounds good to me! I have updated the files accordingly.